scripts/skills/injury/rf_heat_stroke_injury.nut

```diff
@@ -0,0 +1,51 @@
+this.rf_heat_stroke_injury <- this.inherit("scripts/skills/injury/injury", {
+	m = {
+		FatigueCostMult = 1.25,
+	},
+	function create()
+	{
+		this.injury.create();
+		this.m.ID = "injury.rf_heat_stroke";
+		this.m.Name = "Heat Stroke";
+		this.m.Description = "An intense heat caused the player\'s body temperature to soar to dangerous levels.";
```

Comment:
`Being exposed to intense heat caused this character\'s body temperature to soar to dangerous levels.`